app/controllers/clinics_controller.rb

```diff
@@ -25,6 +25,9 @@ def create
   end
 
   def new
+    # i18n-tasks-use t('mongoid.attributes.clinic.phone')
+    # i18n-tasks-use t('mongoid.attributes.clinic.fax')
+    # i18n-tasks-use t('mongoid.attributes.clinic.active')
```

Comment:
are these doing something clever?

Reply:
sort of! It's a signal to `i18n-tasks` to not complain about these i18n translations not being triggered in the code.